e2e-tests/gef/cypress/integration/unissued-facility-change-from-preview-MIN.spec.js

```diff
@@ -0,0 +1,254 @@
+import relative from './relativeURL';
+
+const CONSTANTS = require('../fixtures/constants');
+
+const { add, format } = require('date-fns');
+
+import {
+  MOCK_APPLICATION_MIN, MOCK_FACILITY_ONE, MOCK_FACILITY_TWO, MOCK_FACILITY_THREE, MOCK_FACILITY_FOUR, MOCK_USER_MAKER,
+} from '../fixtures/MOCKS/MOCK_DEALS';
+import applicationPreview from './pages/application-preview';
+import unissuedFacilityTable from './pages/unissued-facilities';
+import aboutFacilityUnissued from './pages/unissued-facilities-about-facility';
+import CREDENTIALS from '../fixtures/credentials.json';
+import applicationSubmission from './pages/application-submission';
+import statusBanner from './pages/application-status-banner';
+
+let dealId;
+let token;
+let facilityOneId;
+
+const unissuedFacilitiesArray = [
+  MOCK_FACILITY_ONE,
+  MOCK_FACILITY_THREE,
+  MOCK_FACILITY_FOUR,
+];
+
+const today = new Date();
+const oneMonth = add(today, { months: 1 });
+const oneMonthDay = format(oneMonth, 'dd');
+const oneMonthMonth = format(oneMonth, 'M');
+const oneMonthYear = format(oneMonth, 'yyyy');
+const twoMonths = add(today, { months: 2 });
+const twoMonthsDay = format(twoMonths, 'dd');
+const twoMonthsMonth = format(twoMonths, 'M');
+const twoMonthsYear = format(twoMonths, 'yyyy');
+const threeMonthsOneDay = add(today, { months: 3, days: 1 });
+const threeMonthsOneDayDay = format(threeMonthsOneDay, 'dd');
+const threeMonthsOneDayMonth = format(threeMonthsOneDay, 'M');
+const threeMonthsOneDayYear = format(threeMonthsOneDay, 'yyyy');
+
+/*
+  for changing facilities to issued from preview page.
+  To unlock functionality, need to first issue one facility from unissued-facility table
+*/
+context('Unissued Facilities MIN - change to issued from preview page', () => {
+  before(() => {
+    cy.apiLogin(CREDENTIALS.MAKER).then((t) => {
+      token = t;
+    }).then(() => {
+      cy.apiCreateApplication(MOCK_USER_MAKER, token).then(({ body }) => {
+        dealId = body._id;
+        cy.apiUpdateApplication(dealId, token, MOCK_APPLICATION_MIN).then(() => {
+          cy.apiCreateFacility(dealId, 'CASH', token).then((facility) => {
```

Comment:
can we change the `'CASH'` to a constant? - the same thing applies for the facilities listed below

Reply:
done